### ISSUE TITLE
Unable to fulfill modified order. Stuck in "Payment Settled"

### ISSUE THREAD

**Issue by vmatekole** [vendure-ecommerce/vendure#846]
1. Order was modified by removing a line item.
2. Refund was correctly applied.

Order is now in "Payment Settled" state. However, I am unable to fulfill order now as the button is unavailable.
When trying to manually change the state of the order to "Shipped" or "Partially shipped". I receive the following error:

https://www.dropbox.com/s/yxes0itu8gp1c4o/Screenshot%202021-04-27%20at%2019.58.24.png?dl=0

Screen shots:

Order
https://www.dropbox.com/s/0a95jon7cnjbfzu/Screenshot%202021-04-27%20at%2019.59.09.png?dl=0
Partial Refund
https://www.dropbox.com/s/klk43ym86hyrs08/Screenshot%202021-04-27%20at%2019.59.43.png?dl=0

**Comment by michaelbromley** [comment]
Thanks for the report. The error originates from here: https://github.com/vendure-ecommerce/vendure/blob/26c1b7a2dbfe5aeb8275f45b7d73cf9f0484127e/packages/core/src/service/helpers/order-state-machine/order-state-machine.ts#L133

which in turn is evaluating this function on each non-cancelled OrderItem: 

https://github.com/vendure-ecommerce/vendure/blob/d3b0f606ed5071419ddd6cb052de8fffb195de9e/packages/core/src/service/helpers/utils/order-utils.ts#L85-L87

I'd be interested to see the output of the following query on the order in question:

```gql
{
  order(id: <id of order>) {
    state
    lines {
      items {
        id
        cancelled
        fulfillment {
          state
        }
      }
    }
  }
}
```

**Comment by vmatekole** [comment]
@michaelbromley  Let me know when this is resolved. We would like to clear these orders from being "Open"

Here is the output:

```json {
  "data": {
    "order": {
      "state": "PaymentSettled",
      "lines": [
        {
          "items": [
            {
              "id": "337",
              "cancelled": true,
              "fulfillment": null
            }
          ]
        },
        {
          "items": [
            {
              "id": "338",
              "cancelled": false,
              "fulfillment": null
            }
          ]
        },
        {
          "items": [
            {
              "id": "339",
              "cancelled": false,
              "fulfillment": null
            }
          ]
        }
      ]
    }
  }
}
```

**Comment by michaelbromley** [comment]
Hey,
Just looked into this again and the basic problem (you mentioned originally) is that you are unable to create a Fulfillment due to the button being disabled. There was a bug with this, which I addressed with [this commit](https://github.com/vendure-ecommerce/vendure/commit/fce00c4) and was released as part of v1.0.1. If you have not already updated, I encourage you to do so and let me know if this resolves the issue for you.
